SU-HSDO/suhumsci
# STN-54: Main Navigation

# READY FOR REVIEW

## Summary
This pull request adds behavior changes to the main navigation and page layout:
- The sidebar navigation is hidden at small-medium screen sizes but visible at the large breakpoint
- the mobile nav will include all pages available at small-medium screens
- the desktop nav will only include the top 2 levels of pages available at large screens

## Need Review By (Date)
asap

## Urgency
high

## Steps to Test
1. Build both CSS and JS
2. Run `npm test` to confirm there are no errors
3. Adjust the main navigation to show 5 levels instead of the default 2:
    `Structure > Blocks Layout` and then configure the Main Navigation block in the `menu` region.
4. Confirm that all the navigation meets all acceptance criteria (link in the Jira card)

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
